examples/Consumer/Program.cs

```diff
@@ -65,7 +65,12 @@ public static void Run_Consume(string brokerList, List<string> topics, Cancellat
                 {
                     // Since a cooperative assignor (CooperativeSticky) has been configured, the
                     // partition assignment is incremental (adds partitions to any existing assignment).
-                    Console.WriteLine($"Incremental partition assignment: [{string.Join(", ", partitions)}]");
+                    Console.WriteLine(
+                        "Partitions incrementally assigned: [" +
```

Comment:
Next time please use an individual commit for this since it is not related to the RC bump